wallet/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -857,7 +857,8 @@ deriveSafeCopySimple 0 'base ''WebTypes.CHash
 deriveSafeCopySimple 0 'base ''WebTypes.CId
 deriveSafeCopySimple 0 'base ''WebTypes.Wal
 deriveSafeCopySimple 0 'base ''WebTypes.Addr
-deriveSafeCopySimple 0 'base ''BackupPhrase
+deriveSafeCopySimple 0 'base ''Entropy
```

Comment:
this can/should be removed (both Entropy and Mnemonic). See comment on the left

Reply:
Will do! Thanks for investigating this.